src/conftest.bash

```diff
@@ -49,7 +49,7 @@ get_rego_namespaces() {
 
   # Safety check to make sure nothing above silently failed
   if [[ ${#namespaces[*]} -lt 1 ]]; then
-    fail "# FATAL-ERROR: (conftest.bash): Found no namespaces; either in the 'policy' dir or matched the regex '${regex}'"
+    fail "# FATAL-ERROR: (conftest.bash): Found no namespaces; either in the 'policy' dir or matched the regex '${regex}'" || return $?
   fi
```

Comment:
` || return $?` will this just fail the execution if the response code from the command was greater than 0 or was your ambition to catch it and move on?

Reply:
yep, it will return the error code, which it always does